apps/judicial-system/web/src/routes/Shared/Cases/Cases.tsx

```diff
@@ -204,7 +204,6 @@ export const Cases: React.FC = () => {
       caseToDelete.state === CaseState.SUBMITTED ||
       caseToDelete.state === CaseState.RECEIVED
     ) {
-      await sendNotification(caseToDelete.id, NotificationType.REVOKED)
```

Comment:
Consider removing `sendNotification` and `NotificationType` from the imports above as they are no longer used.